Gems/Terrain/Assets/Shaders/Terrain/ClipmapComputeHelpers.azsli

```diff
@@ -229,10 +241,15 @@ BilinearUvs CalculateClipmapUv(
     return uvs;
 }
 
-float4 ColorPointSampling(Texture2DArray<float4> colorClipmap, uint2 uv, uint clipmapLevel)
+float4 ColorPointSampling(Texture2DArray<float4> colorClipmap, uint2 uv, uint clipmapLevel, float debugEnabled)
 {
     uint3 texelIndex = uint3(uv, clipmapLevel);
-    return colorClipmap[texelIndex];
+    float4 color = colorClipmap[texelIndex];
+    // Debug option
+    float3 overlayColor = ClipmapLevelDebugColor[min(clipmapLevel, 4)];
```

Comment:
just an idea, you could always do clipmapLevel % 5 to get colors on all levels, if you're ok with colors repeating.

Reply:
That's a good idea.